src/diff/index.js

```diff
@@ -134,6 +134,8 @@ export function diff(dom, parentDom, newVNode, oldVNode, context, isSvg, excessD
 			let vnode = c._prevVNode = coerceToVNode(c.render(c.props, c.state, c.context));
 			c._dirty = false;
 
+			if (options.afterRender) options.afterRender();
```

Comment:
Wondering if we could use `options.diffed` instead. It's callsite is just a couple lines later at the end of `diff()`.